Add lksctp-tools and libsctp-dev packages

jamonation opened the pull request:
Useful to have this library for any higher layer protocols or applications that use SCTP e.g. [WebRTC](https://webrtcforthecurious.com/docs/07-data-communication/) and [Diameter](https://www.f5.com/glossary/diameter-protocol) for IP telephony respectively.

### Pre-review Checklist

#### For new package PRs only
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates

jamonation committed:
add lksctp-tools and libsctp-dev

Signed-off-by: Jamon Camisso <jamonation+git@gmail.com>

vaikas commented on lksctp-tools.yaml:
```diff
@@ -0,0 +1,58 @@
+      - uses: split/dev
+    description: libsctp dev
+
+update:
```
They do not seem to use releases, so you probably have to use use-tags for the update section.
```
  github:
    use-tag: true
```

fwiw, you can test the update section locally like this:
```
GITHUB_TOKEN="<YOUR TOKEN HERE> wolfictl check update  --override-version 0.0.1 <yourfile>
```

jamonation replied:
Ah nice, TIL!

jamonation committed:
Add standalone lib and docs subpackages to lksctp-tools

Signed-off-by: Jamon Camisso <jamonation+git@gmail.com>

jamonation committed:
Merge branch 'main' into lksctp-tools

jamonation commented:
@vaikas if you're inclined to review this again, I added a couple sub-packages. One for docs, and another for shared library itself. I also fixed the auto-update portion with the `use-tag` magic.

jamonation committed:
Merge branch 'main' into lksctp-tools
